streams/src/test/java/org/apache/kafka/streams/state/internals/CompositeReadOnlyKeyValueStoreTest.java

```diff
@@ -74,9 +74,11 @@ public void before() {
                 Serdes.String())
                 .build();
 
-        store.init(new InternalMockProcessorContext(new StateSerdes<>(ProcessorStateManager.storeChangelogTopic("appId", storeName), Serdes.String(), Serdes.String()),
-                                                    new NoOpRecordCollector()),
-                store);
+        final InternalMockProcessorContext context = new InternalMockProcessorContext(new StateSerdes<>(ProcessorStateManager.storeChangelogTopic("appId", storeName),
```

Comment:
Can this be switched to `MockInternalProcessorContext`?

Reply:
There is another PR that tries to consolidate `MockInternalProcessorContext` and `InternalMockProcessorContext` so I intentionally leave it as not changed: https://github.com/apache/kafka/pull/7718